### ISSUE TITLE
Doesn't seem towrite non-standard fields

### ISSUE THREAD

**Issue by Pumpapa** [dylang/node-rss#22]
My feed contains non-standard fields such as mediavalue. They aren't written in the feed by default. Should I inject them in a namespace and if so, which, and how?

**Comment by SachaG** [comment]
The readme is a bit confusing on this… First it mentions `custom_namespaces` and `custom_elements`, but then the examples use `customNamespaces` and `custom`. 

And it's not clear if you can add any value as a custom element, or it has to follow the `namespace:element` syntax? In any case I haven't been able to get this to work yet :(

**Comment by maxnowack** [comment]
`custom_namespaces` and `custom_elements` are correct.
`custom_namespaces` can be specified as an object:
* key: namespace name (without `xmlns:`)
* value: namespace url

`custom_elements` takes an array of objects in [node-xml](https://github.com/dylang/node-xml) syntax

@SachaG can you post your code?

**Comment by SachaG** [comment]
Thanks, that helps. But part of the problem is that I'm not sure what the `custom_namespaces` URL is supposed to point to. 

Basically, all I want to do is add an `imageUrl` field to my RSS items. So I have something like:

```
var RSS = require('rss');
var feed = new RSS({
    title: 'title',
    description: 'description',
    custom_namespaces: {
      'imageUrl': 'http://example.com'
    },
});

feed.item({
    title:  'Item Name',
    description: 'description',
    custom_elements: [
      { 'imageUrl': myImageUrl}
    ]
});
var xml = feed.xml("\t");
console.log(xml);
```

But I have no clue what "http://example.com" should be?

**Comment by SachaG** [comment]
It turned out I was running a very old version of the package, one that obviously didn't support `custom_namespaces` and `custom_elements`. I updated and everything works! Sorry about that :)

**Comment by ErisDS** [comment]
Closing this as it is working as of #35 being merged (version 1.1.0)
